third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompting-checker.js

```diff
@@ -293,8 +294,8 @@ UserPromptingChecker.prototype.storeUserSelection_ =
   this.localStoragePromiseResolver_.promise.then(
```

Comment:
(I'm too stupid to add the comment where it actually belongs)
Do we need to explicitly ignore garbage values in 
|contents| in localStorageLoadedCallback_ now (if written by a buggy previous version)?

Reply:
Yes, that's a good suggestion, we should not crash due to anything in the externally stored data. I was actually planning to do this, but in a follow-up change.